src/ui/blocks/Arithmetic.hpp

```diff
@@ -1,16 +1,17 @@
-#include "arithmetic_block.h"
+#ifndef OPENDIGITIZER_ARITHMETIC_HPP
+#define OPENDIGITIZER_ARITHMETIC_HPP
 
-#include <type_traits>
+#include <gnuradio-4.0/Block.hpp>
 
-#include "../flowgraph.h"
+namespace opendigitizer {
 
 template<typename T>
     requires std::is_arithmetic_v<T>
-struct MathNode : public gr::Block<MathNode<T>> {
-    gr::PortIn<T>                           in1{};
-    gr::PortIn<T>                           in2{};
+struct Arithmetic : public gr::Block<Arithmetic<T>> {
```

Comment:
generally OK for this PR but we should move all of these isolated block implementation to the GR 4.0 repo and also make them conform and check the corresponding items in summary issue [EPIC: List of GR 3.10 blocks & Others to be ported to GR 4.0](https://github.com/fair-acc/graph-prototype/issues/161)